action.yaml

```
@@ -0,0 +1,13 @@
+name: Setup go env using go version from go.mod
```

// Let's keep names concise and use description for further description
```suggestion
name: Setup Go Environment
```